tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -284,31 +294,52 @@ private set
         /// </summary>
         [IgnoreForSnapshot] // We don't record this in telemetry currently, but if we do, we'll record it when we set its
         internal MetricsTransportType MetricsTransport { get; private set; }
+
+        /// <summary>
+        /// Gets or sets the agent host to use when <see cref="MetricsTransport"/> is <see cref="TransportType.UDP"/>
+        /// </summary>
+        [IgnoreForSnapshot] // We don't record this in telemetry currently, but if we do, we'll record it when we set its
```

Comment:
Its what? the suspense is killing me

Reply:
Dunno, whatever it is that we'll do with `MetricsTransport` too

![image](https://github.com/DataDog/dd-trace-dotnet/assets/18755388/14a740d7-f832-4099-bbb9-ecc00f17d008)

Reply:
FWIW, fixed the comment typo in both in the stacked PR
- https://github.com/DataDog/dd-trace-dotnet/pull/5224